### PR TITLE
Python gitbot

### PR DESCRIPTION
**Pull request by RuiSanches**
- Creates a discord based python bot, that listens to messages triggered by a github webhook, on a discord channel.
- By reading the message, the bot extracts the pr url associated with the github action and triggers a jenkins build.
- Upon receiving the results of the job on jenkins, notifies the success or failure of the job and the correspondent jenkins url.

### CODE REVIEW

File: triggers/jenkins/Jenkinsfile.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+pipeline:
```
Comment (Pedro-Sanches) [inline triggers/jenkins/Jenkinsfile.yml]: For now I would move this file to jenkins/Jenkinsfile.yml. You could leave gitbot_ci.sh where it stands if you want, you'll just have to change line 10. This is not ideal, but for now, at least the pipeline will run. We will have to create a task to improve the bit of using the right Jenkinfile.yml

**Comment by Pedro-Sanches** [comment]
Excellent work with gitbot

**Review by Pedro-Sanches — APPROVED** [review]

**Comment by RuiSanches** [comment]
Test

**Comment by RuiSanches** [comment]
Test

**Comment by RuiSanches** [comment]
Test

**Comment by RuiSanches** [comment]
Test

**Comment by RuiSanches** [comment]
Test

**Comment by RuiSanches** [comment]
Test

**Comment by RuiSanches** [comment]
Test

**Comment by RuiSanches** [comment]
Test

**Comment by RuiSanches** [comment]
Test

**Comment by RuiSanches** [comment]
TEst

**Comment by RuiSanches** [comment]
Test

**Pull request merged**